common/buildings/wc_special_buildings.txt

```diff
@@ -60,12 +60,53 @@ gurubashi_arena_01 = {
 
 	type = special
 }
+########
+# Icecrown Citadel
+
+icecrown_citadel_01 = {
+	type_icon = "icon_structure_icecrown_citadel.dds"
+	
+	province_modifier = {
+		fort_level = 7
+	}
+
+	character_modifier = {
+		levy_reinforcement_rate = 0.1
+		levy_size = 0.1
+		martial = 2
+		monthly_dread = 0.1
+		vassal_limit = 5
+		vassal_levy_contribution_mult = 0.1
+		taiga_advantage = 4
```

Comment:
I would add winter bonuses here too.

Reply:
there is a taiga advantage, winter bonuses are applied to maa's directly AFAIK

Reply:
Nah, there's a way to add winter everything to character too. Used in the winter trait.
![image](https://user-images.githubusercontent.com/46384992/216828071-27464c20-ff94-410c-9d33-7b94a403f78a.png)